src/cache/lru_cache_manager/lru_queue.h

```diff
@@ -63,10 +65,10 @@ LRUNode *dequeue(LRUQueue *queue);
  * @brief  Enqueues a new node with given key and value 
  * @param  *queue: LRU Queue address (pointer)
  * @param  *hashKey: New node's key - charecter array in size of HASH_KEY_LENGTH
- * @param  resultset: New node's value
+ * @param  cacheValue: New node's value
  * @retval Newly genereted LRU Node, which is in the tail of the LRU queue
  */
-LRUNode *enqueue(LRUQueue *queue, const char *hashKey, ResultSet* resultset);
+LRUNode *enqueue(LRUQueue *queue, unsigned long long const key, void *cacheValue);
```

Comment:
Consider defining a new type hash_key_t as unsigned long long
to make this a bit more generic replace cacheValue with value.

Reply:
done